documentation/sphinx/source/api-c.rst

```diff
@@ -480,6 +486,12 @@ Applications must provide error handling and an appropriate retry loop around th
 
    |future-return0| the estimated size of the key range given. |future-return1| call :func:`fdb_future_get_int64()` to extract the size, |future-return2|
 
+.. function:: FDBFuture* fdb_transaction_get_range_split_points( FDBTransaction* tr, uint8_t const* begin_key_name, int begin_key_name_length, uint8_t const* end_key_name, int end_key_name_length, int64_t chunkSize)
```

Comment:
`chunkSize` is using camel case here while all the other variables are not.

Reply:
Done.